<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" verbose="0" name="Failed suite [SuiteName]" parallel="tests">
  <test name="Login Test_2(failed)" parallel="tests">
    <classes>
      <class name="com.unisys.pom.testcases.LoginTest">
        <methods>
          <include name="quit"/>
        </methods>
      </class> <!-- com.unisys.pom.testcases.LoginTest -->
      <class name="com.unisys.pom.testcases.LoginTest_2">
        <methods>
          <include name="doLogin_2" invocation-numbers="0"/>
          <include name="quit"/>
        </methods>
      </class> <!-- com.unisys.pom.testcases.LoginTest_2 -->
      <class name="com.unisys.pom.testcases.LoginTest_1">
        <methods>
          <include name="quit"/>
        </methods>
      </class> <!-- com.unisys.pom.testcases.LoginTest_1 -->
    </classes>
  </test> <!-- Login Test_2(failed) -->
</suite> <!-- Failed suite [SuiteName] -->
